lib/smart_app_launch/well_known_capabilities_stu2_test.rb

```diff
@@ -10,7 +10,18 @@ class WellKnownCapabilitiesSTU2Test < Inferno::Test
       If the `sso-openid-connect` capability is supported, then `issuer` and `jwks_uri` must be
       present. If `sso-openid-connect` capability is not supported, then `issuer` must be omitted.
     )
- 
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@372',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@373',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@374',
```

Comment:
@374 is well covered in lib/smart_app_launch/well_known_endpoint_test.rb and doesn't seem properly tested here.

@374
> Responses for `/.well-known/smart-configuration` requests SHALL be JSON, regardless of `Accept` headers provided in the request.

Reply:
Removed in [00d865f](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/105/commits/00d865f61fa250db6f0e76c9ea30a521676586a4).